common.pri

```diff
@@ -2,6 +2,10 @@
 #
 # To use it, define ROOT_DIR variable and include($$ROOT_DIR/common.pri)
 
+# We use some library features that were introduced in Mac OS X 10.8.
+# Qt5.6.x sets target OS X version to 10.7 which leads to compile errors.
+# TODO: Remove when qt5.7 is available in brew.
```

Comment:
Why do we need Qt5.7 from brew? It's possible to install it directly, without brew.